Adding vpn/whitelist information on FAQ and Troubleshooting pages (#2696)
cypress-io/cypress-documentation: source/guides/references/troubleshooting.md

{% imgTag /img/api/debug/failed-log.png "cypress-failed-log terminal output" %}

> ## Add some subdomains on your VPN withelist for the Dashboard work properly

Reviewer: ```suggestion
## Add some subdomains on your VPN whitelist for the Dashboard to work properly
```

> You have to add the following subdomains on your vpn whitelist to the Dashboard work properly:

Reviewer: ```suggestion
You have to add the following subdomains on your VPN whitelist for the Dashboard to work properly:
```

> - Cypress API: https://**API**.cypress.io

Reviewer: ```suggestion
- Cypress API: https://**api**.cypress.io
```
